index.js

```diff
@@ -71,6 +71,11 @@ function ignoreAuth(req, res) {
         useCreateIndex: true
     })
 
+    var p = 'loopback'
+    if (config.get('trustedProxy')) {
+        p += ',' + config.get('trustedProxy');
```

Comment:
It might be better to write this as:

```javascript
if (config.get('trustedProxy')) {
	app.set('trust proxy', ['loopback', ...config.get('trustedProxy')].join(','))
}
```

And then turn the trustedProxy config option from a string to an array.  Our config can have complex objects in it.

Reply:
fixed